src/chains-config/metadata-consts/substrateConsts.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * WEIGHT NOTES (POLKADOT | KUSAMA)
+ * @constant extrinsicBaseWeight
+ * @constant blockWeights
+ *
+ * The following weights are used for Polkadot and Kusama chains. Both are using
+ * the same fee calculations currently, and share the same Integer values. Because
```

Comment:
Is this not true for `Westend` also?

Reply:
Yea thats a good point we can add it in as well for Westend. The original thought was I found that early versions of Westend didn't support fee calculations. So I didn't add it in.

Reply:
I just have to map out the runtimes, but will have this added in the next 30 mins.

Reply:
I think we actually should not add Westend. I don't want to establish the expectation that we will always support Westend with max optimizations. The block weight store still works with westend, we just won't hardcode values on startup so the first query in an unseen runtime will be slow but subsequent queries in that runtime will be fast - this should save us some surface area for manual maintenance.

Reply:
Added!